src/server.js

```diff
@@ -0,0 +1,12 @@
+const http = require('http');
+const console = require('console');
```

Comment:
`require('console');` what is this :eyes: ??

Reply:
It's a eslint magic 😂 
When we used console.log() in server.listen() function, eslint suggested that we require console!